console/src/components/Services/Data/TableBrowseRows/ViewRows.js

```diff
@@ -763,9 +764,7 @@ const ViewRows = ({
           <b className={styles.padd_small_right}>Selected:</b>
           {selectedRows.length}
           <button
```

Comment:
Use the common Button component.

Reply:
Isn't it duplicated work, since we'll be soon integrating buttons from the design system?